### DISCUSSION TITLE
sql.json - filter parameter missing?

### DISCUSSION THREAD

**Discussion by vitalyster** [porsager/postgres#406] [Q&A] [answered]
`JSON.stringify` have second `filter` parameter, what about `sql.json`? Any different way to transform object before writing to the database?

**Comment by porsager** [comment, marked as answer]
sql.json isn't necessary for v3. Just use regular Javascript ;)

```js
sql`
  insert into x (some_json) values (${
    someArray.filter(x => x.some_prop === 'wat')
  }::jsonb)
`
```
